src/engine/source/hlp/src/LogQLParser.cpp

```diff
@@ -10,7 +10,8 @@ static const std::unordered_map<std::string_view, ParserType> kECSParserMapper {
     { "source.ip", ParserType::IP },
     { "server.ip", ParserType::IP },
     { "source.nat.ip", ParserType::IP },
-    { "timestamp", ParserType::Any },
+    { "timestamp", ParserType::Ts },
+    { "JSON", ParserType::JSON},
```

Comment:
JSON parser type was migrated to kTempTypeMapper on a previous PR.

Reply:
Done!